pkg/gadgets/internal/socketenricher/bpf/sockets-map.h

```diff
@@ -77,6 +77,7 @@ struct sockets_value {
 	__u64 uid_gid;
 	char task[TASK_COMM_LEN];
 	__u64 sock;
+	__u64 ipv6only;
```

Comment:
Do we really need 64bits for this? Can't we move this as the last element and use a bitfield in case we want to other small fields in the future?